pages/linux/autopkgtest.md

```diff
@@ -0,0 +1,24 @@
+# autopkgtest
+
+> Run tests on Debian packages.
+> More information: <https://wiki.debian.org/ContinuousIntegration/autopkgtest>.
+
+- Build the package in the current directory and run all tests using the `null` virtualization server:
+
+`autopkgtest -- null`
```

Comment:
Is the `null` bit here something the user can change (e.g. is it a hostname to SSH into or something)?

Reply:
`null` here means that the user chose to run the tests directly on their system, as opposed to running them inside a  `chroot`, `lxc`, `qemu`...

Reply:
I think we should explain that in the description. Something like:
`- Build the package in the current directory and run all tests directly on the system:`
And then, since `null` is an argument, we should put in in `{{}}`:
```suggestion
- Build the package in the current directory and run all tests using the `null` virtualization server:

`autopkgtest -- {{null}}`
```